lib/testsuite/index.js

```diff
@@ -260,6 +260,13 @@ class TestSuite {
       return null;
     }
 
+    Object.defineProperty(global, 'app', {
```

Comment:
same as above

Reply:
The above is not applicable here since `browser` property is added to `global` object somewhere else.